src/allmydata/test/web/test_web.py

```diff
@@ -1035,6 +1035,122 @@ def _check_retrieve(res):
 
         return d
 
+    def test_status_path_error(self):
+        # Expect an error, because path is expected to be of the form
+        # "/status/{up,down,..}-%number", with a hyphen.
```

Comment:
Maybe this could be a docstring instead.

Reply:
Updated in 6a2ce9e.